Documentation/FrameworkOverview.md

```diff
@@ -162,6 +192,9 @@ of [signal operators][RACSignal+Operations] instead of blocking work.
 [RACScheduler][] is also somewhat similar to `NSOperationQueue`, but schedulers
 do not allow tasks to be reordered or depend on one another.
 
+<!-- TODO: Update? Is this te place to talk about 
+PropertyType / Parameterized types / Objective-C bridging? -->
+
```

Comment:
`PropertyType` should probably be mentioned immediately before or after `Action`.